test/jest-unit.config.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Any reason to have this empty file here?

Reply:
Ah, was assuming that something somewhere was assuming the presence of this file, turns out not. Deleted